tools/src/main/java/org/infinispan/tools/store/migrator/StoreMigrator.java

```diff
@@ -20,25 +44,64 @@
 public class StoreMigrator {
 
    private static final int DEFAULT_BATCH_SIZE = 1;
+   // A list of all internal classes that were previously marshallable, but we no longer provide
+   // guarantees over byte compatibility in future versions
+   private static final Set<Class<?>> INTERNAL_BLACKLIST = new HashSet<>();
```

Comment:
I'm not sure I would have read the store migrator changes anyway, but it would have been nice it they were in a separate PR, because GitHub collapses all of them.

Reply:
Agreed, it does introduce a lot of noise. I'll issue the commit as a separate PR once this is merged.